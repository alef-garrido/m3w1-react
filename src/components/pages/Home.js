import React from 'react';
import './home.css';

function Home() {
  const paragraph = `
  Math magicians is a demo site that uses REACT.js 
  to render a Single-Page-Application. We used Math 
  as a topic to create components that allow users to 
  interact with a calculator and make basic operations, 
  or read math-related quotes. Each component is displayed 
  on its own "page". Please have fun and inspect the app to 
  find possible improvements! You can comment and collaborate 
  using Github's repo: 
  https://github.com/alef-garrido/react-spa-math-magicians. 
  Hope you like it!
    `;
  return (
    <div className="home--container">
      <h2>Welcome to our page</h2>
      <p>{paragraph}</p>
    </div>
  );
}

export default Home;
