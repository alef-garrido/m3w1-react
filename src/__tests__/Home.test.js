import renderer from 'react-test-renderer';
import Home from '../components/pages/Home';

it('renders correctly', () => {
  const tree = renderer
    .create(<Home />)
    .toJSON();
  expect(tree).toMatchSnapshot();  
});