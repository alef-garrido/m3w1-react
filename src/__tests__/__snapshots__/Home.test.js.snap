// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="home--container"
>
  <h2>
    Welcome to our page
  </h2>
  <p>
    
  Math magicians is a demo site that uses REACT.js 
  to render a Single-Page-Application. We used Math 
  as a topic to create components that allow users to 
  interact with a calculator and make basic operations, 
  or read math-related quotes. Each component is displayed 
  on its own "page". Please have fun and inspect the app to 
  find possible improvements! You can comment and collaborate 
  using Github's repo: 
  https://github.com/alef-garrido/react-spa-math-magicians. 
  Hope you like it!
    
  </p>
</div>
`;
